src/pages/library/DataBrowserFeedbackModal.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonPrimary, IdContainer, spinnerOverlay } from 'src/components/common'
+import { TextArea, TextInput } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import { Ajax } from 'src/libs/ajax'
+import { FormLabel } from 'src/libs/forms'
+import * as Utils from 'src/libs/utils'
+import validate from 'validate.js'
+
+
+export const DataBrowserFeedbackModal = ({ onSuccess, onDismiss, primaryQuestion, sourcePage }) => {
+  const [contactEmail, setContactEmail] = useState('')
+  const [feedback, setFeedback] = useState('')
+  const [submitting, setSubmitting] = useState(false)
+
+  const submit = Utils.withBusyState(setSubmitting, async () => {
+    await Ajax().Surveys.submitForm('1FAIpQLSevEVLKiLNACAsti8k2U8EVKGHmQ4pJ8_643MfdY2lZEIusyw',
+      { 'entry.477992521': feedback, 'entry.82175827': contactEmail, 'entry.367682225': sourcePage })
+    onSuccess()
+  })
+
+  const errors = validate({ feedback, contactEmail },
+    { feedback: { length: { minimum: 1, maximum: 2000 } }, contactEmail: contactEmail ? { email: true } : {} })
+
+  return h(Modal, {
+    onDismiss,
+    shouldCloseOnEsc: false,
+    width: 500,
+    title: 'Give feedback',
+    okButton: h(TooltipTrigger, { content: !!errors && _.map(error => div({ key: error }, [error]), errors) },
+      [h(ButtonPrimary, {
+        disabled: errors,
+        onClick: submit
+      }, ['Submit'])]
+    )
+  }, [
+    h(IdContainer, [id => h(Fragment, [
+      h(FormLabel, { required: true, htmlFor: id, style: { fontSize: 14, fontWeight: 300 } },
```

Comment:
Does this label need the extremely light font weight? I don't think anything else in Terra is below 400. Also, could this label and the "Can we contact you with further questions?" label be styled consistently?

Reply:
UX question @lstrano-broad